src/lib/common/sol-missing.h

```diff
@@ -98,3 +98,7 @@ accept4(int sockfd, struct sockaddr *addr, socklen_t *len, int flags)
 #ifndef I2C_RDRW_IOCTL_MAX_MSGS
 #define I2C_RDRW_IOCTL_MAX_MSGS 42
 #endif
+
+#if !HAVE_SYS_AUXV_H
+#define AT_EXECFN  31   /* filename of program */
```

Comment:
If `sys/auxv.h` isn't available, then it's likely that `getauxval()` isn't as well. Not sure if this is correct.